forge/test/models/pytorch/text/t5/test_t5.py

```
@@ -95,16 +104,18 @@ def forward(self, decoder_input_ids, encoder_outputs):
     pytest.param(
         "google/flan-t5-small",
         id="google_flan_t5_small",
-        marks=[pytest.mark.push],
+        marks=[pytest.mark.push, pytest.mark.xfail(reason="Tensor mismatch. PCC = 0.897340042137896")],
```

Let's not use xfails for nighlty tests. This will provide invalid data in the DB.

If tests fails, let it fail for now :))

---

xfail present in test_stable_diffusion_xl too. so I removed that along with this now.